src/stylesheets/main.scss

```diff
@@ -168,6 +168,74 @@ $mq-breakpoint-widescreen: 1200px;
   }
 }
 
+$colour-list-breakpoint: 980px;
+
+.app-c-colour-list{
```

Comment:
Nit: selectors should have a space after them before the opening curly brace

Reply:
do we have a linter?

Reply:
We noticed earlier this week that the linter is not looking at main.scss so it's a bit wild west at the moment

Reply:
can I run it manually somehow?

Reply:
Let's not block this PR and sort linting issues after this lands